### PR TITLE
APIS-5254 - Fix tests

### PR DESCRIPTION
**Pull request by AndySpaven**
Fixing a broken test revealed acceptance tests that have been broken for some indeterminate period of time - the on(Page) method was faulty.

### CODE REVIEW

File: acceptance/acceptance/specs/ApiGatekeeperApplicationSpec.scala [hunk 1]
```diff
@@ -111,27 +108,33 @@ class ApiGatekeeperApplicationSpec extends ApiGatekeeperBaseSpec with StateHisto
...
+      stubApplicationForDeveloper(unverifiedUser.userId)
 
       When("I select to navigate to a collaborator")
       ApplicationsPage.selectDeveloperByEmail(unverifiedUser.email)
 
       Then("I am successfully navigated to the developer details page")
       on(DeveloperDetailsPage)
+
+      println("*****************************************")
+      println(DeveloperDetailsPage.bodyText)
```
[the remark this answers is not in the document]
Comment (peteslater-ee) [reply acceptance/acceptance/specs/ApiGatekeeperApplicationSpec.scala:120]: No

File: acceptance/acceptance/specs/ApiGatekeeperApplicationSpec.scala [hunk 2]
```diff
@@ -111,27 +108,33 @@ class ApiGatekeeperApplicationSpec extends ApiGatekeeperBaseSpec with StateHisto
...
     }
   }
 
   def stubDeveloper() = {
-    val encodedEmail = URLEncoder.encode(unverifiedUser.email, "UTF-8")
-
-    stubFor(get(urlEqualTo(s"""/developer?email=$encodedEmail"""))
-      .willReturn(aResponse().withStatus(OK).withBody(unverifiedUserJson)))
+    println(s"**** ${unverifiedUser.userId.value}")
```
[the remark this answers is not in the document]
Comment (peteslater-ee) [reply acceptance/acceptance/specs/ApiGatekeeperApplicationSpec.scala:125]: Cheers

File: acceptance/acceptance/specs/ApiGatekeeperApplicationSpec.scala [hunk 3]
```diff
@@ -111,27 +108,33 @@ class ApiGatekeeperApplicationSpec extends ApiGatekeeperBaseSpec with StateHisto
...
+    stubFor(
+      get(urlPathEqualTo("/developer"))
+      // .withQueryParam("developerId", matching(".*"))
```
[the remark this answers is not in the document]
Comment (peteslater-ee) [reply acceptance/acceptance/specs/ApiGatekeeperApplicationSpec.scala:128]: Thanks

File: acceptance/acceptance/NavigationSugar.scala [hunk 4]
```diff
@@ -34,16 +33,27 @@ trait NavigationSugar extends WebBrowser with Eventually with Assertions with Ma
   }
 
   def go(page: WebLink)(implicit webDriver: WebDriver) = {
-    goo to page
+    WebBrowser.go.to(page)
```
Comment (mattgreenuk) [inline acceptance/acceptance/NavigationSugar.scala:36]: Great code

File: app/controllers/ApplicationController.scala [hunk 5]
```diff
@@ -89,8 +89,10 @@ class ApplicationController @Inject()(val applicationService: ApplicationService
         val defaults = Map("page" -> "1", "pageSize" -> "100", "sort" -> "NAME_ASC")
         val params = defaults ++ request.queryString.map { case (k, v) => k -> v.mkString }
 
+        Logger.warn("POMEGRANATE - before")
```
[the remark this answers is not in the document]
Comment (peteslater-ee) [reply app/controllers/ApplicationController.scala:92]: Grrr

File: app/controllers/ApplicationController.scala [hunk 6]
```diff
@@ -89,8 +89,10 @@ class ApplicationController @Inject()(val applicationService: ApplicationService
...
         for {
           par <- applicationService.searchApplications(env, params)
+          _ = Logger.warn("POMEGRANATE - here")
```
[the remark this answers is not in the document]
Comment (peteslater-ee) [reply app/controllers/ApplicationController.scala:95]: Argh

**Review by mattgreenuk — APPROVED** [review]

**Pull request merged**